src/app/components/MostRead/index.tsx

```diff
@@ -100,6 +110,15 @@ const MostRead = ({
       </MostReadSection>
     ) : null;
 
+  /**
+   * If renderCanonicalOnAmp is true, then use the canonical version of the component instead of the AMP version.
+   *
+   * This is to prevent double fetching of most read data.
+   */
```

Comment:
Just checking that this is something we want to do for the MostRead page? Will AMP not cache the MostRead page for awhile if we render the Canonical version? The client side fetch is needed on the AMP version to ensure the data isn't stale.

This may be a non-issue anymore, but just wanted to check.

Reply:
Ah yes I can see the old `serverRenderOnAmp` flag. Yea if it replicates the existing logic then its all good. Thanks for double checking!